libuserhelper.cpp

```diff
@@ -388,3 +411,32 @@ QStringList LibUserHelper::groups(uint uid)
 
     return rv;
 }
+
+QString LibUserHelper::getUserUUID(uint uid) const
+{
+    struct lu_error *error = nullptr;
+    struct lu_context *context = lu_start(NULL, lu_user, NULL, NULL, NULL, NULL, &error);
+    if (!context) {
+        qCWarning(lcSUM) << "Error creating context:" << lu_strerror(error);
+        lu_error_free(&error);
+        return {};
+    }
+
+    struct lu_ent *ent = lu_ent_new();
+
+    QString userUuid;
+    if (lu_user_lookup_id(context, uid, ent, &error)) {
+        userUuid = lu_ent_get_first_string(ent, LU_GECOS);
```

Comment:
Doesn't this return the whole gecos field like "name,uuid"? Name needs to stripped.

Reply:
Yes, need to strip.